### PR TITLE
Add ambient sounds to more stations

### PR DESCRIPTION
**Pull request by roadrunner56**
**Content (Artwork / Missions / Jobs)**

## Summary
On Derpy's trello board it was mentioned that a lot of stations, especially those newly added to the game, don't have an ambient station track when they probably should. I added an ambient track to various industrial and refinery stations, as well as some abandoned stations. Please let me know if some of these additions aren't appropriate for a station.

## Save File
Any save file with a jump drive works.

### CODE REVIEW

**Comment by RisingLeaf** [comment]
I will request a review from the leads of the affected factions just to make sure.

**Comment by Saugia** [comment]
I don't believe that's necessary, all stations should have this sound, and if it should be different, changed appropriately.

**Comment by RisingLeaf** [comment]
I dont see why all stations should have this sound, it would be strange for all to sound the same, so I think it is better to check this with some people

**Comment by roadrunner56** [comment]
Please note that I didn't add it to every station, only those that have some sort of industry or are otherwise abandoned but still have an atmosphere. I didn't add it to the nice Hai retirement stations, for example.

**Comment by RisingLeaf** [comment]
oh, ok, that makes more sense
I should read more carefully sorry

File: data/map planets.txt [hunk 1]
```diff
@@ -347,6 +347,7 @@ planet "Blue Interior"
 planet Bluerun
 	attributes hai station uninhabited
 	landscape land/station33
+	music ambient/machinery
 	description `This station was mothballed sometime in the last sixty thousand years. The docking area has an atmosphere, and a status panel indicates that the station's generators are functional but shut down, with standby mechanisms running on solar power alone.`
```
Comment (warp-core) [inline data/map planets.txt:351]: I'm not sure this description suggests we should hear much from this station, certainly not more noise than our own ship might make.

File: data/map planets.txt [hunk 2]
```diff
@@ -748,6 +749,7 @@ planet Darkmetal
 planet Darkrest
 	attributes hai station uninhabited
 	landscape land/station34
+	music ambient/machinery
 	description `This station was mothballed sometime in the last sixty thousand years. The docking area has an atmosphere, and a status panel indicates that the station's generators are functional but shut down, with standby mechanisms running on solar power alone.`
```
Comment (warp-core) [inline data/map planets.txt:753]: Maybe we should just leave sounds on these stations until they have proper descriptions?

File: data/map planets.txt [hunk 3]
```diff
@@ -1019,6 +1022,7 @@ planet "Ef Osch"
 planet Eldergrasp
 	attributes hai retirement station uninhabited
 	landscape land/station32
+	music ambient/machinery
 	description `A visitor to this station will immediately notice that the gravity is lower than most planets by about a third. There is a generously appointed reception area with an orderly on duty who informs you that this is a respite home for the elderly, and unless you have family here, you will not be permitted to wander the station.`
 	spaceport `The spaceport is virtually non-existent, with just a single cafeteria, a fuel pump for the desperate, and a dedicated loading area for loading and unloading the facility's regular deliveries. Despite its small size, everything is in excellent condition.`
```
Comment (warp-core) [inline data/map planets.txt:1027]: Should this station really have this sound?

File: data/map planets.txt [hunk 4]
```diff
@@ -3690,6 +3721,7 @@ planet Splashdown
 planet "Ssil Vida"
 	attributes emitter "remnant station" sheragi uninhabited
 	landscape land/station11
+	music ambient/machinery
 	description `The silence of this station rings in your ears, and as you explore you are increasingly surprised that it even has an atmosphere. It is obviously ancient, and patches of ambiguous green and purple material resembling fungus grow through the walls. Whoever built this station designed it for beings much larger than you. Either that, or they liked spacious environments. Even the shortest doorway you pass through is easily double your height. They are always wider than they are tall.`
```
Comment (warp-core) [inline data/map planets.txt:3725]: This one *definitely* shouldn't have sound, at least, not to begin with.

File: data/map planets.txt (hunk 3 above)
Comment (Saugia) [reply data/map planets.txt:1027]: It still has gravity and houses people, so it should, yes. If we're concerned about the *specific* sound, we can worry about that later.

File: data/map planets.txt [hunk 5]
```diff
@@ -2144,6 +2157,7 @@ planet "Laki Nemparu"
 planet Letis'sei
 	attributes ka'het uninhabited
 	landscape land/loc3
+	music ambient/machinery
```
Comment (Saugia) [inline data/map planets.txt:2160]:
```suggestion
```
Becca said this one shouldn't have a sound. The other two Het stations are fine.

File: data/map planets.txt (hunk 3 above)
Comment (roadrunner56) [reply data/map planets.txt:1027]: Actually, now that I think of it, this station shouldn't really have this sound. It's not industrial or a military outpost, and the description says that it's well-maintained.

File: data/map planets.txt (hunk 4 above)
Comment (roadrunner56) [reply data/map planets.txt]: Yeah I don't really know what I was thinking with this one.

**Review by Saugia — APPROVED** [review]

**Pull request merged**